wordpress-mobile/WordPress-Android
# Enable plugins for self-hosted sites

Fixes #14856 

Related FluxC changes - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2043

This PR adds support for plugins to self-hosted sites. One thing that doesn't work yet is the "Auto-update" feature on plugins on self-hosted sites. The API doesn't support it but we don't have a way to disable it right now in the app.

To test:
- Log in with a self-hosted site
- Go to My Site 
- Notice the "Plugins" item is now available
- Go to "Plugins"
- Try installing/updating/deleting plugins and check everything works as expected

## Regression Notes
1. Potential unintended areas of impact
- The whole plugins feature
- Gutenberg network functionality

2. What I did to test those areas of impact (or what existing automated tests I relied on)
- All the tests are in FluxC

PR submission checklist:
